### PR TITLE
[MIG] Migrate project_closing

### PR DESCRIPTION
**Pull request by charbeljc**

### CODE REVIEW

File: project_closing/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+##############################################################################
+{
+    "name": "Project closing",
+    "version": "1.1",
+    "author": "Camptocamp",
+    "website": "http://www.camptocamp.com",
+    "category": "project Management",
+    "depends": ["project"],
+    "description": """
+    Automatic account analytic closing when related project is closed.
```
Comment (pedrobaeza) [inline project_closing/__openerp__.py]: Please extract this to a README.rst file and follow OCA's template

File: project_closing/project.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+from osv import orm
+
+
+class project_project(orm.Model):
```
Comment (pedrobaeza) [inline project_closing/project.py]: Use CamelCase for class naming

File: project_closing/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    and If a projet is open, the related analytic account will be  re-open.
```
Comment (pedrobaeza) [inline project_closing/__openerp__.py]: s/projet/project

Comment (pedrobaeza) [inline project_closing/__openerp__.py]: Double space between `be` and `re-open`

File: project_closing/project.py [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    _inherit = 'project.project'
+
+    def set_done(self, cr, uid, ids, context=None):
```
Comment (pedrobaeza) [inline project_closing/project.py:8]: If it's not too much to ask, please switch to new API

File: project_closing/__openerp__.py (hunk 3 above)
Comment (gurneyalex) [inline project_closing/__openerp__.py]: s/re-open/re-opened/

s/If/if/
s/is open/is opened/

**Comment by charbeljc** [comment]
[Build Status](https://travis-ci.org/charbeljc/project-service)

File: project_closing/README.rst [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+`here <https://github.com/OCA/{project_repo}/issues/new?body=module:%20{module_name}%0Aversion:%20{version}%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+
+Credits
+=======
+
+Contributors
+------------
+
+* FIXME
```
Comment (dreispt) [inline project_closing/README.rst]: Your name can go here

File: project_closing/__openerp__.py [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    "name": "Project closing",
+    "version": "1.1",
+    "author": "Camptocamp",
+    "website": "http://www.camptocamp.com",
```
Comment (dreispt) [inline project_closing/__openerp__.py:25]: Add license key

File: project_closing/__openerp__.py [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+    "category": "project Management",
```
Comment (dreispt) [inline project_closing/__openerp__.py]: Uppercase "Project"

**Comment by dreispt** [comment]
I must say I'm surprised that there is a `state` field on `project.project` when we already have one on `account.analytic.account`. This ought to be a design bug...

Other than some small notes, :+1:

File: project_closing/__openerp__.py (hunk 6 above)
Comment (charbeljc) [inline project_closing/__openerp__.py:25]: done.

File: project_closing/__openerp__.py (hunk 7 above)
Comment (charbeljc) [inline project_closing/__openerp__.py]: done.

**Pull request closed without merging**